src/celestia/qt/qtglwidget.cpp

```diff
@@ -147,6 +148,24 @@ static GLContext::GLRenderPath getBestAvailableRenderPath(const GLContext& glc)
 
 void CelestiaGlWidget::initializeGL()
 {
+    GLenum glewErr = glewInit();
+    if (glewErr != GLEW_OK)
+    {
+        QString rn;
+        switch(glewErr) {
```

Comment:
According to https://codeyarns.com/2015/09/14/how-to-check-error-in-glew/ GLEW uses only 4 errors (I'm too lazy to recheck myself.)

Reply:
That's right indeed.

Reply:
`switch` doesn't use Celestia's formatting.